assets/externalized/scripts/enums.lua

```diff
@@ -1,41 +1,322 @@
-Items = {
-    SUNGOGGLE = 212,
-    CANTEEN   = 214,
-}
+--[[
+ This file was auto-generated. Do not manually edit.
+--]]
 
-Visibility = {
-	BURIED               =  -3,
-	HIDDEN_IN_OBJECT     =  -2,
-	INVISIBLE            =  -1,
-	VISIBLE              =   1,
+SectorFlag = {
+	SF_USE_MAP_SETTINGS = 1,
+	SF_ENEMY_AMBUSH_LOCATION = 2,
+	SF_PLAYER_KNOWS_ENEMIES_ARE_HERE = 4,
+	SF_ALREADY_VISITED = 32,
+	SF_USE_ALTERNATE_MAP = 64,
+	SF_PENDING_ALTERNATE_MAP = 128,
+	SF_ALREADY_LOADED = 256,
+	SF_HAS_ENTERED_TACTICAL = 512,
+	SF_SKYRIDER_NOTICED_ENEMIES_HERE = 1024,
+	SF_HAVE_USED_GUIDE_QUOTE = 2048,
+	SF_SMOKE_EFFECTS_TEMP_FILE_EXISTS = 1048576,
+	SF_LIGHTING_EFFECTS_TEMP_FILE_EXISTS = 2097152,
+	SF_REVEALED_STATUS_TEMP_FILE_EXISTS = 16777216,
+	SF_DOOR_STATUS_TEMP_FILE_EXISTS = 33554432,
+	SF_ENEMY_PRESERVED_TEMP_FILE_EXISTS = 67108864,
+	SF_CIV_PRESERVED_TEMP_FILE_EXISTS = 134217728,
+	SF_ITEM_TEMP_FILE_EXISTS = 268435456,
+	SF_ROTTING_CORPSE_TEMP_FILE_EXISTS = 536870912,
+	SF_MAP_MODIFICATIONS_TEMP_FILE_EXISTS = 1073741824,
+	SF_DOOR_TABLE_TEMP_FILES_EXISTS = 2147483648,
```

Comment:
How does this even work? The limitations you linked say:
> Enum value must be in range [MAGIC_ENUM_RANGE_MIN, MAGIC_ENUM_RANGE_MAX].    By default MAGIC_ENUM_RANGE_MIN = -128, MAGIC_ENUM_RANGE_MAX = 128.

Is this actually about the count of enum values? You're not overriding the range like with some others.

Reply:
It uses `magic_enum::flags` so the handling should be different,